py/src/ai/h2o/sparkling/ExternalBackendConf.py

```diff
@@ -192,5 +192,5 @@ def setExternalExtraJars(self, paths):
         return self
 
     def setExternalCommunicationCompression(self, compression):
-        self._jconf.setExternalCommunicationCompression(self, compression)
+        self._jconf.setExternalCommunicationCompression(compression)
```

Comment:
Good catch, thanks!

Reply:
It would be good to write tests for all configurations so we avoid these mistakes. I think we already have jira for it, but that is not scope of this PR :)